index.html

```diff
@@ -3001,112 +3001,359 @@ <h3>Truncating or limiting the length of strings</h3>
-	    </tr>
-		<tr>
-			<td rowspan=2>Han</td>
-			<td rowspan=2>14</td>
-			<td rowspan=2>3</td>
-			<td><kbd lang="zh">在最美丽的城镇，那里的房屋�</kbd></td>
-		</tr>
-		<tr>
-			<td><code>E5 9C A8 E6 9C 80 E7 BE 8E E4 B8 BD E7 9A 84 E5 9F 8E E9 95 87 EF BC 8C E9 82 A3 E9 87 8C E7 9A 84 E6 88 BF E5 B1 8B <span style="color:red">E5</span></code></td>
-	    </tr>
-		<tr>
-			<td rowspan=2>Emoji</td>
-			<td rowspan=2>10
-			<td rowspan=2>4</td>
-			<td><kbd>🙊🙁😢😠😧😎😽😉😄😮</kbd></td>
-		</tr>
-		<tr>
-			<td><code>F0 9F 99 8A F0 9F 99 81 F0 9F 98 A2 F0 9F 98 A0 F0 9F 98 A7 F0 9F 98 8E F0 9F 98 BD F0 9F 98 89 F0 9F 98 84 F0 9F 98 AE</code></td>
-		</tr>
-		</table>
+        <p>Finally, notice that the Tamil word takes more room with four [=visual text units=] than an English word with nine:</p>
+        
+       <p class="bigtext">யூனிகோட்</p>
+       <p class="bigtext">Memorable</p>
     </aside>
 
-
-	<div class="req" id="char_trunc_units">
-	<p class="advisement">Specifications that limit the length of a string MUST specify which type of unit (extended grapheme clusters, Unicode code points, or code units) the length limit uses.</p>
-	</div>
-
+ 
+    
 	<div class="req" id="char_trunc_unit_rec">
-	<p class="advisement">Specifications that limit the length of a string SHOULD specify the length in terms of Unicode code points.</p>
+	<p class="advisement">Specifications that limit the maximum allowable length of a string SHOULD specify the length in terms of [=Unicode code points=].</p>
 	</div>
-
-	<div class="req" id="char_trunc_byte_boundary">
-	<p class="advisement">If a specification sets a length limit in code units (such as bytes), it MUST specify that truncation can only occur on code point boundaries.</p>
+    
+    <p>Most limits are actually related to storage limitations, such as the size of a database field or a length limit in a protocol. Such limits are expressed either in terms of [=code points=] in Unicode or in terms of [=code units=] (such as bytes) in a specific [=character encoding=]. [=Code points=] provides the best user experience, since all Unicode code points are treated identically: if text is truncated after 40 code points, all languages and scripts get the same number of code points to work with. By contrast, when the size limit is expressed in [=code units=], such as bytes in UTF-8, users who write in a language that mostly uses ASCII letters get many more characters (code points) for a given size limit than user's whose language is mostly made up of characters that take 2-, 3-, or 4-bytes per code point.</p>
+    
+    <div class="req" id="char_trunc_grapheme_boundary">
+	   <p class="advisement">Specifications that limit the length of a string SHOULD require truncation on [=visual text unit=] boundaries (usually as approximated by [=grapheme cluster=] boundaries). Truncation in the middle of a visual text unit such as a <a>combining character sequence</a> can alter the meaning of the string.</p>
```

Comment:
```suggestion
	   <p class="advisement">Specifications that require truncation of strings SHOULD truncate the text at [=visual text unit=] boundaries (usually as approximated by [=grapheme cluster=] boundaries). Truncation in the middle of a visual text unit such as a <a>combining character sequence</a> can alter the meaning of the string.</p>
```

Reply:
The second sentence "Truncation in the middle..." is actually an explanation, so should be dropped into the start of the explanatory text that follows, rather than included in the mustard.

Reply:
This was as far as i could get with my review for this week.

Reply:
I did a deeper revision, in part to address the second comment.